backend/auth-service/cp-trace/catchpoint-config-staging.yml

```diff
@@ -0,0 +1,19 @@
+catchpoint:
```

Comment:
Remove env config file. There should be only one config file for production which might be `catchpoint-config.yml`